packages/driver/src/cy/commands/connectors.coffee

```diff
@@ -41,7 +41,10 @@ module.exports = (Commands, Cypress, cy, state, config) ->
     remoteSubject = cy.getRemotejQueryInstance(subject)
 
     args = remoteSubject or subject
-    args = if subject?._spreadArray then args else [args]
+
+    try
+      hasSpreadArray = "_spreadArray" in subject or subject?._spreadArray
```

Comment:
This works for proxies or regular objects, but I don't love it. Would love any ideas for alternatives!